studio/components/interfaces/Integrations/IntegrationConnection.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  IntegrationConnection,
+  IntegrationConnectionProps,
+} from 'components/interfaces/Integrations/IntegrationPanels'
+import ConfirmationModal from 'components/ui/ConfirmationModal'
+import { useIntegrationsVercelConnectionSyncEnvsMutation } from 'data/integrations/integrations-vercel-connection-sync-envs-mutation'
+import { IntegrationProjectConnection } from 'data/integrations/integrations.types'
+import { BASE_PATH } from 'lib/constants'
+import { useRouter } from 'next/router'
+import React, { useCallback, useState } from 'react'
+import { Button, Dropdown, IconChevronDown, IconLoader, IconRefreshCw, IconTrash, Modal } from 'ui'
+
+interface IntegrationConnectionItemProps extends IntegrationConnectionProps {
+  onDeleteConnection: (connection: IntegrationProjectConnection) => void | Promise<void>
+}
+
+const IntegrationConnectionItem = React.forwardRef<HTMLLIElement, IntegrationConnectionItemProps>(
```

Comment:
can we import forwardRef from React, rather than importing the whole React here

Reply:
done